docs/development/submitting-patches.rst

```diff
@@ -57,6 +57,61 @@ Important information should be documented with prose in the ``docs`` section.
 To ensure it builds and passes `doc8`_ style checks you can run
 ``make docs`` and ``make lint`` respectively.
 
+Keeping your local branch updated
+---------------------------------
+
+As you work, it is helpful to keep your local branch up-to-date with the PRs
+that are merged into the main Warehouse repository. Most people working on
+the project use this workflow.
```

Comment:
The wording here implies that you're keeping a single branch up-to-date with multiple branches (since each PR is a branch). I suggest you revise to say something about keeping your local *master* branch up-to-date with the master branch in the main Warehouse repository, which moves forward as maintainers merge pull requests.

Reply:
I figure you're still working on [this](https://github.com/pypa/warehouse/pull/2760#discussion_r160774503) - just a reminder.